cmd/verify/config.go

```diff
@@ -214,3 +225,35 @@ func parseConfig() {
 		}
 	}
 }
+
+// TransportConfigS3 is a helper method to setup TLS for the S3 client.
+func TransportConfigS3(c *Config) *http.Transport {
+	cfg := new(tls.Config)
+
+	// Enforce TLS1.2 or higher
+	cfg.MinVersion = 2
+
+	// Read system CAs
+	var systemCAs, _ = x509.SystemCertPool()
+	if reflect.DeepEqual(systemCAs, x509.NewCertPool()) {
+		log.Debug("creating new CApool")
+		systemCAs = x509.NewCertPool()
+	}
+	cfg.RootCAs = systemCAs
+
+	if c.S3Archive.Cacert != "" {
+		cacert, e := ioutil.ReadFile(c.S3Archive.Cacert) // #nosec this file comes from our configuration
+		if e != nil {
+			log.Fatalf("failed to append %q to RootCAs: %v", cacert, e)
+		}
+		if ok := cfg.RootCAs.AppendCertsFromPEM(cacert); !ok {
+			log.Debug("no certs appended, using system certs only")
+		}
+	}
+
+	trConfig := &http.Transport{
+		TLSClientConfig:   cfg,
+		ForceAttemptHTTP2: true}
+
+	return trConfig
+}
```

Comment:
This could be placed in the storage.go file.

Reply:
I don't really care where this is placed. But we should probably have a generic function to init tlsConfigs for everything, but that's a separate PR.

Reply:
Ok, let's fix this later!